backup_and_restore/application_backup_and_restore/oadp-performance/oadp-recommended-network-settings.adoc

```diff
@@ -0,0 +1,23 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="oadp-recommended-network-settings.adoc"]
+= {oadp-short} recommended network settings
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+
+:context: oadp-recommended-network-settings
+
+toc::[]
+
+For a supported experience with {oadp-first}, you should have a stable and resilient network across {OCP-short} nodes, S3 storage, and in supported cloud environments.
+
+To ensure successful backup and restore operations, Red Hat recommends that your network settings meet the following minimum requirements:
+
+* Bandwidth: Greater than 2 MB
+* Packet loss: Range 1% to 5%
+* Packet corruption: Range 1% to 5%
+* Latency: Range 100ms (±20ms)
+
```

Comment:
Lets also add about Jitter, I suppose we have some data regarding that as well

Reply:
@shubham-pampattiwar Since jitter refers to the the variation in the interval between successive data packets, should we include it as just a note, or add a recommended range for it as well?
FYI @weshayutin and @TzahiAshkenazi

Reply:
Incorporated based on Tzahi's inputs.